README.md

```diff
@@ -1,127 +1,194 @@
-- [Stylesheet guidelines](docs/Stylesheet/README.md) - Contains rules for CSS/SASS code style
-- [JavaScript guidelines](docs/JavaScript/README.md) - Contains rules for JS/JSX code style
-- [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-- [e2e and performance testing docs](e2e_tests/README.md) - Contains documents for create and running e2e and performance tests
+![CircleCI](https://img.shields.io/circleci/build/github/binary-com/deriv-app) ![Prerequisite](https://img.shields.io/badge/node-%3E%3D12.8.0-blue.svg) ![Prerequisite](https://img.shields.io/badge/npm-%3E%3D6.9.0-blue.svg) [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
+![Sonar Tech Debt](https://img.shields.io/sonar/tech_debt/DerivFE_deriv-app?server=https%3A%2F%2Fsonarcloud.io)
+![Sonar Violations (short format)](https://img.shields.io/sonar/violations/DerivFE_deriv-app?server=https%3A%2F%2Fsonarcloud.io)
+
+**In this document**:
+
+-   [Other Documents](#other-documents)
+-   [Pre-installation](#Pre-installation)
+-   [Quick start](#Quick-start)
+-   [Packages](#packages)
+-   [Working With This Repo](#working-with-this-repo)
+    -   [Package names](#package-names)
+    -   [Usage](#usage)
+        -   [Starting a Development Server](#starting-a-dev-server)
+        -   [How to Clean Packages](#how-to-clean-packages)
+        -   [Examples of Script Usage](#examples-of-script-usage)
+        -   [Release](#release)
+-   [PR Guidelines](#pr-guidelines)
+-   [FAQ](#faq)
+
+## Other documents:
+
+-   [General](docs/README.md) - Contains general philosophy and overview of this package
+-   [Stylesheet guidelines](docs/Stylesheet/README.md) - Contains rules for CSS/SASS code style
+-   [JavaScript guidelines](docs/JavaScript/README.md) - Contains rules for JS/JSX code style
+-   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
+-   [e2e and performance testing docs](e2e_tests/README.md) - Contains documents for create and running e2e and performance tests
 
 [comment]: <> (TODO: Refactor Clean Project to be under usage)
 
-## Installation
-In order to work on your own version of the Deriv application, please **fork this project**.
+## Pre-installation
+
+Before running or contribute to this project, you need to have the setup of the following package in your environment.
+
+-   node >=12.3.0
+-   npm >=6.9.0
+-   git (for `contribution`)
+
+## Quick start
+
+1.  **Fork the project**
+
+    In order to work on your own version of the Deriv application, please fork the project to your own repository.
+
+2.  **Clone using SSH**
+
+    ```sh
+    git clone git@github.com:binary-com/deriv-app.git
+    ```
+
+3.  **Enter project directory**
+
+    ```sh
+    cd deriv-app
+    ```
 
-You will need to perform the following on your development machine:
+4.  **Install your dependencies:**
 
```

Comment:
In some circumstances this script not working, because there is no package-lock on the root folder. ( for instance after cleaning the package ).
In that case user should run `npm i` in the root folder of project to regenerate the package-lock file.